<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RankingDialog</class>
 <widget class="QDialog" name="RankingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>排行榜</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDialog { background-color: #0f172a; border-radius: 20px; }

#labelTitle { color: white; font-size: 28px; font-weight: bold; }

#closeButton {
    background: transparent;
    color: #94a3b8;
    font-size: 28px;
    font-weight: bold;
    border: none;
}
#closeButton:hover { color: #ef4444; }

#labelRank1, #labelRank2, #labelRank3 {
    color: black;
    font-weight: bold;
    font-size: 16px;  
    border-radius: 16px;
    padding: 14px 20px;
    background-color: #4338CA;
}
#labelRank1 { background-color: #009fd4; }
#labelRank2 { background-color: #00BCD4; }
#labelRank3 { background-color: #B2EBF2; }
   </string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>排行榜</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="topSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignmentFlag::AlignHCenter">
    <widget class="QLabel" name="labelRank1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>295</width>
       <height>65</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>295</width>
       <height>65</height>
      </size>
     </property>
     <property name="text">
      <string>NO.1 暂无用户数据</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignmentFlag::AlignHCenter">
    <widget class="QLabel" name="labelRank2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>295</width>
       <height>65</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>295</width>
       <height>65</height>
      </size>
     </property>
     <property name="text">
      <string>NO.2 暂无用户数据</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignmentFlag::AlignHCenter">
    <widget class="QLabel" name="labelRank3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>295</width>
       <height>65</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>295</width>
       <height>65</height>
      </size>
     </property>
     <property name="text">
      <string>NO.3 暂无用户数据</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="indent">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="bottomSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
